windows-installer.xml

```diff
@@ -0,0 +1,431 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.nuxeo.packaging</groupId>
+  <artifactId>windows-installer-3parties</artifactId>
+  <version>3.1</version>
+  <packaging>zip</packaging>
+  <description>3rd parties to package within Windows installer</description>
+
+  <url>http://www.nuxeo.com/en/products</url>
+  <organization>
+    <name>Nuxeo</name>
+    <url>http://www.nuxeo.com</url>
+  </organization>
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+    </license>
+  </licenses>
+
+
+  <mailingLists>
+    <mailingList>
+      <name>Nuxeo ECM list</name>
+      <post>ecm@lists.nuxeo.com</post>
+      <subscribe>http://lists.nuxeo.com/mailman/listinfo/ECM</subscribe>
+      <unsubscribe>http://lists.nuxeo.com/mailman/listinfo/ECM</unsubscribe>
+      <archive>http://lists.nuxeo.com/pipermail/ecm/</archive>
+      <otherArchives>
+        <otherArchive>
+          http://dir.gmane.org/gmane.comp.cms.nuxeo.bugs
+        </otherArchive>
+        <otherArchive>
+          http://www.mail-archive.com/ecm@lists.nuxeo.com/
+        </otherArchive>
+      </otherArchives>
+    </mailingList>
+    <mailingList>
+      <name>Nuxeo ECM checkins list</name>
+      <subscribe>
+        http://lists.nuxeo.com/mailman/listinfo/ecm-checkins
+      </subscribe>
+      <unsubscribe>
+        http://lists.nuxeo.com/mailman/listinfo/ecm-checkins
+      </unsubscribe>
+      <archive>http://lists.nuxeo.com/pipermail/ecm-checkins/</archive>
+      <otherArchives>
+        <otherArchive>
+          http://dir.gmane.org/gmane.comp.cms.nuxeo.scm
+        </otherArchive>
+        <otherArchive>
+          http://www.mail-archive.com/ecm-checkins@lists.nuxeo.com/
+        </otherArchive>
+      </otherArchives>
+    </mailingList>
+    <mailingList>
+      <name>Nuxeo ECM developers list</name>
+      <post>nuxeo-dev@lists.nuxeo.com</post>
+      <subscribe>http://lists.nuxeo.com/mailman/listinfo/nuxeo-dev</subscribe>
+      <unsubscribe>
+        http://lists.nuxeo.com/mailman/listinfo/nuxeo-dev
+      </unsubscribe>
+      <archive>http://lists.nuxeo.com/pipermail/nuxeo-dev/</archive>
+    </mailingList>
+  </mailingLists>
```

Comment:
some deprecated URLs, either fix or don't mention (+JIRA ticket) ; see 
- https://doc.nuxeo.com/corg/quality-assurance-and-continuous-integration/
- https://nuxeowiki.atlassian.net/wiki/spaces/NIG/pages/2621552
- https://groups.google.com/a/nuxeo.com/forum/#!myforums